apps/admin_app/lib/admin_app_web/helpers.ex

```diff
@@ -122,30 +158,35 @@ defmodule AdminAppWeb.Helpers do
     end
   end
 
-  def order_xlsx_exporter(user) do
-    orders = Repo.all(Order) |> Repo.preload([:line_items])
+  def xlsx_exporter(user, type) do
+    data_list =
+      case type do
+        "order" ->
+          Repo.all(Order) |> Repo.preload([:line_items])
```

Comment:
Moreover, these methods to get all the orders and products should go in product models not here. We have to expose a method from domain and use them here in the application. 

Think of it as this way. If we ever need to write a custom logic in the core app which accesses this logic then we have to certainly write this logic again. there is no way we'll be able to reuse this custom order/product logic to get all products and orders. 

Please pay more attention to DRY and SRP when you are coding. This is and will be violating both the principles.